backend-db/modules/db/couchdb/index.js

```diff
@@ -612,6 +624,92 @@ const layer = {
         }
         return false;
     },
+
+    //========================================================
+    // TEAMS
+    //========================================================
+    getTeam: async id => {
+        return await adapter.getOneEntityById('Team', '_design/all/_view/view', id);
+    },
+    getAllTeams: async () => {
+        const ret = await adapter.getEntities('Team', '_design/all/_view/view', {sorted: false});
+        return ret;
+    },
+    getRawTeamDoc: async id => {
+        return await adapter.getOneRawDocById('Team', '_design/all/_view/view', id);
+    },
```

Comment:
You can get rid of async and await, since there's only a one-liner that returns a promise anyway.